.travis.yml

```diff
@@ -50,29 +52,12 @@ env:
   global:
   - DEADLINKS_VERS=0.3.0
   - SODIUM_VERS=1.0.13
```

Comment:
I guess we don't need this variable anymore since we install whatever version is in the PPA.